eth/api.go

```diff
@@ -694,6 +695,12 @@ func (s *PublicBlockChainAPI) doCall(args CallArgs, blockNr rpc.BlockNumber) (st
 	}
 	stateDb = stateDb.Copy()
 
+	// If there's no code to interact with, respond with an appropriate error
+	if args.To != nil {
+		if code := stateDb.GetCode(*args.To); len(code) == 0 {
+			return "0x", nil, bind.ErrNonExistentContract
```

Comment:
All in all it seemed a bit large of a dependency to pull in the entire eth protocol just to handle a 3 word error string.